src/components/TimeSeries/TimeSeriesForm.scss

```diff
@@ -0,0 +1,19 @@
+.time-series-form {
```

Comment:
Use Tailwind classes instead of defining custom styles using SCSSS. I've already created a utility class for entering items using flex-box in `src/scss/_layout.scss` file. You can use `flex-center` class and add the remaining styles using appropriate classes from Tailwind.

  If need to use these classes multiple times and don't want to repeat yourself, you can create the SCSS file in the `src/scss` directory e.g. in the new `_components.scss` file but I prefer to use Tailwind wherever it's possible.